### PR TITLE
Changed the webView in Sponsors

### PR DESCRIPTION
**Pull request by RutsSantos**
#### Description:
Changed the webView for a openURL so now the web pages open in the default browser

#### Linked issues:
[Ticket 222](https://trello.com/c/Ebh6t4Xj/222-cambiar-los-webviews-de-auspiciadores-por-un-redirect-al-browser-por-default-del-dispositivo-1)

#### How to test:
Tap on a sponsor card

### CODE REVIEW

**Review by JoseFcoEstrella — APPROVED** [review]

**Review by Faustrox — COMMENTED** [review]
There are some code about dialogs and stuff related in `Details.js`... Can you delete it? It's not necessary now

**Review by D-Rosa99 — CHANGES_REQUESTED** [review]
check `app/views/DR/News/Details.js` for dialogs component that we do not need any more.

I don't know if you have to delete the resources, because there're no longer be use, check that out too

https://github.com/intellisysdcorp/covid-safe-paths/pull/118/files
This is the PR where is all

File: app/views/DR/News/Details.js [hunk 1]
```diff
@@ -19,18 +20,12 @@ const fixerImage = `
...
+    params: { source, switchScreenTo = 'WebView' },
   },
 }) => {
   navigation.setOptions({ headerShown: false });
 
   const { t } = useTranslation();
-  const [timer, setNewTimer] = useState(null);
   const [showDialog, setShowDialog] = useState(false);
   const [dialogText, setDialogText] = useState('');
   const closeDialog = () => {
```
Comment (Faustrox) [inline app/views/DR/News/Details.js:31]: Delete this

File: app/views/DR/News/Details.js [hunk 2]
```diff
@@ -70,32 +65,17 @@ const Details = ({
...
-              }, 30000),
-            );
-          }}
-          onLoadEnd={() => {
-            clearTimeout(timer);
-          }}
+          renderLoading={() => activityIndicatorLoadingView()}
           onError={() => {
             setDialogText(t('label.dialog_error_advice'));
             setShowDialog(true);
```
Comment (Faustrox) [inline app/views/DR/News/Details.js:81]: You can delete this

Comment (RutsSantos) [reply app/views/DR/News/Details.js:81]: is good to have an error handler, since the news open in a webView

**Review by Faustrox — APPROVED** [review]
LGTM

**Review by D-Rosa99 — APPROVED** [review]

**Pull request merged**